### PR TITLE
Fix Portuguese (Portugal) strings

### PR DESCRIPTION
**Pull request by luisfcorreia**
Align strings with release

### CODE REVIEW

File: Marlin/language_pt.h [hunk 1]
```diff
@@ -18,20 +18,20 @@
...
 #define MSG_PREHEAT_PLA_ALL                 "Pre-aq. PLA Tudo"
 #define MSG_PREHEAT_PLA_BEDONLY             "Pre-aq. PLA " LCD_STR_THERMOMETER "Base"
-#define MSG_PREHEAT_PLA_SETTINGS            "PLA definicoes"
+#define MSG_PREHEAT_PLA_SETTINGS            "Definicoes PLA"
 #define MSG_PREHEAT_ABS                     "Pre-aquecer ABS"
 #define MSG_PREHEAT_ABS_N                   "Pre-aquecer ABS "
 #define MSG_PREHEAT_ABS_ALL                 "Pre-aq. ABS Tudo"
 #define MSG_PREHEAT_ABS_BEDONLY             "Pre-aq. ABS " LCD_STR_THERMOMETER "Base"
-#define MSG_PREHEAT_ABS_SETTINGS            "ABS definicoes"
+#define MSG_PREHEAT_ABS_SETTINGS            "Definicoes PLA"
```
Comment (KiteLab) [inline Marlin/language_pt.h]: ??? PLA ???

File: Marlin/language_pt.h [hunk 2]
```diff
@@ -54,8 +54,8 @@
 #define MSG_MIN                             LCD_STR_THERMOMETER " Min"
 #define MSG_MAX                             LCD_STR_THERMOMETER " Max"
 #define MSG_FACTOR                          LCD_STR_THERMOMETER " Fact"
-#define MSG_AUTOTEMP                        "Autotemp"
-#define MSG_ON                              "On "
```
Comment (KiteLab) [inline Marlin/language_pt.h:58]: This space is intentional. to delete the second f of "off"

File: Marlin/language_pt.h (hunk 1 above)
Comment (luisfcorreia) [inline Marlin/language_pt.h]: sorry, a bad case of copy/paste.

Please apologise

**Comment by AnHardt** [comment]
The `RC` branch is closed for new direct commits. Adapted this to `RCBugFix` in #2700.

**Pull request closed without merging**